third_party/repositories.bzl

```diff
@@ -258,7 +258,7 @@ def xchammer_dependencies():
     namespaced_git_repository(
         name = "Tulsi",
         remote = "https://github.com/bazel-ios/tulsi.git",
-        tag = "rules_ios-0.0.1",
+        commit = "72adb78843fb6d04088984d889bf634ca1948d7d",
```

Comment:
Shouldn't we point this to the tag?

https://github.com/bazel-ios/tulsi/commits/rules_ios-5.0.0

Reply:
This should mirror what we have in rules_ios, however since we setup the tulsi version in there it shoulnd't be an issue I think: https://github.com/bazel-ios/rules_ios/pull/464/files#diff-7e7efce7dea772315337e8a3a0b7a20dc841f8dfdfd93b6be8fa92d10d8917b6R270